apis/workload/v1alpha1/types.go

```diff
@@ -55,6 +55,9 @@ type KubernetesApplicationSpec struct {
 	// KubernetesApplicationResources.
 	ResourceSelector *metav1.LabelSelector `json:"resourceSelector"`
 
+	// TODO(muvaf): Only MatchLabels field of LabelSelector is used. Incorporate
```

Comment:
Are we going to have TODOs in the code? Shouldn't we just open an issue instead?